fs-storage/src/base_storage.rs

```diff
@@ -24,4 +24,8 @@ pub trait BaseStorage<K, V>: AsRef<BTreeMap<K, V>> {
     /// Remove all persisted data
     /// by pre-configured location in the file-system.
     fn erase(&self) -> Result<()>;
+
+    /// Merge two storages instances
+    /// and write the result to the filesystem.
```

Comment:
Here should be same as in other storage mutations, i.e. we should enable user to control when writes happen.

@twitu should we merge into in-memory model, but not write until the user calls `fn write_fs`?
If we have merging in `fn write_fs` then we'd do merging twice...

Reply:
Or should we just allow the exception for `merge_from` for this moment, and call `write_fs` from it?

Reply:
Now that we've removed `write_fs` from `set`, it'll be consistent to remove it from `merge_from` as well. This way we can leave the writes entirely up to the application consuming ark-rust logic.